summoner-cli/src/Summoner/Template/Cabal.hs

```diff
@@ -211,6 +211,8 @@ cabalFile Settings{..} = File (toString settingsRepo ++ ".cabal") cabalFileConte
             -Wmissing-export-lists
             -Wpartial-fields
             |]
+        <> memptyIfFalse (settingsTestedVersions `hasLeast` Ghc881)
+            "-Wmissing-deriving-strategies\n"
```

Comment:
Should it be error?

Reply:
Also, we should definitely go for the conditional options per GHC versions as you suggested before!

Reply:
I think it should be error. It's an interesting question, to which warnings we should add `-Werror` but I'd love to add it here since strategies are implied by our style-guide and this is a good default for our projects. And yes, I agree that after we introduce common stanzas, a lot of this stuff will be improved 🙂  Thanks for support my idea!